### PR TITLE
Google's policy changes...

### PR DESCRIPTION
**Pull request by MarshCastle**
Also - not checked - but this may positively affect the "Usage Limits" paragraph at the end of this page. As it's all coming into effect now, I thought it worth raising, but a bit pushed for resource on testing it atm.
Sorry :(
Martin

### CODE REVIEW

**Review by homotechsual — CHANGES_REQUESTED** [review]
Looks good - however the Google API keys are supported.

File: docs/initial-set-up/mapping.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ After enabling the mapping provider, you may need to use this script to generate
 [http://wiki.civicrm.org/confluence/display/CRMDOC/Batch+Geocoding+Script](http://wiki.civicrm.org/confluence/display/CRMDOC/Batch+Geocoding+Script)
 
 !!! note "Usage Limits"
-    Some providers have usage limits on their Mapping and Geocoding APIs. If you go over the limits you may receive a overlimit error when trying to save an address. Currently there is no support for integrating the [Google APIs Console license](https://developers.google.com/maps/documentation/javascript/usage#usage_limits) (which is needed for sites whose usage is over limit). Contact info at civicrm dot org if you are interested in helping implement this.
\ No newline at end of file
+    Some providers have usage limits on their Mapping and Geocoding APIs. If you go over the limits you may receive a overlimit error when trying to save an address. Currently there is no support for integrating the [Google APIs Console license](https://developers.google.com/maps/documentation/javascript/usage#usage_limits) (which is needed for sites whose usage is over limit). Contact info at civicrm dot org if you are interested in helping implement this.
```
Comment (homotechsual) [inline docs/initial-set-up/mapping.md:29]: **Suggestion:** change to:

> Some providers have usage limits on their Mapping and Geocoding APIs. If you go over the limits you may receive a over limit error when trying to save an address.

**Comment by John-B** [comment]
It is also worth pointing out in the docs that Google have changed their policy so mapping will no longer work without an API key, and it is impossible to get an API key without enabling billing in your Google account.

**Comment by homotechsual** [comment]
After thinking about this @John-B personally I feel it's down to the user to choose their geocoding/mapping provider and deal with the terms - if we put it into the docs we're commiting ourselves to explaining any future changes. Where do we stop and do we monitor other geocoders/mapping systems to the same extent?

**Comment by homotechsual** [comment]
@marsh-circle are you able to "unstick" this PR as per feedback above?

**Comment by homotechsual** [comment]
Closing in favour of #410

**Pull request closed without merging**